Sources/Danger/File.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+// MARK: - File
+
+/// A simple typealias for strings representing files
+public typealias File = String
+
+extension File {
+
+    public var fileType: FileType? {
+        return FileType(from: self)
+    }
+    
+    public var name: String {
+        return String(self)
+    }
+
+}
+
+// MARK: - FileType
+
+public enum FileType: String, Equatable {
+    case h, json, m, markdown = "md", pbxproj, plist, storyboard, swift
```

Comment:
What do you think about adding the extensions for `.xcworkspace`, `.xcscheme`, `.yml` and `.mm`?

Reply:
Yep, I think that's a good idea 👍